packages/blockchain-extension/extension/commands/UserInputUtil.ts

```diff
@@ -134,14 +134,30 @@ export class UserInputUtil {
 
         const environments: FabricEnvironmentRegistryEntry[] = await FabricEnvironmentRegistry.instance().getAll(showLocalFabric);
 
-        const environmentsQuickPickItems: Array<IBlockchainQuickPickItem<FabricEnvironmentRegistryEntry>> = environments.map((environment: FabricEnvironmentRegistryEntry) => {
-            let label: string = environment.name;
-            if (environment.name === FabricRuntimeUtil.LOCAL_FABRIC) {
-                label = FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME;
-            }
+        let environmentsQuickPickItems: Array<IBlockchainQuickPickItem<FabricEnvironmentRegistryEntry>>;
+        if (showOnlyOps) {
```

Comment:
this is really confusing showOnlyOps should either be true or false you should use the show local  variable to know whether to show the local fabric one or not

Reply:
The reason I had three different conditions was because we could have 3 potential scenarios:
1. Only show Ops Tool environments (filter nodes command)
2. Show every environment except the Ops tool environments (import nodes command)
3. Show every environment in general - incl. Ops (for e.g. deleteNode command)